components/contribution-flow/StepPayment.js

```diff
@@ -64,7 +66,7 @@ const getPaymentMethodMetadata = pm => {
       />
     );
   } else if (pm.type === 'virtualcard') {
-    if (pm.expiryDate) {
+    if (pm.expiryDate && pm.balance > minBalance) {
```

Comment:
Rather than checking `pm.balance` in both `if`, would be cleaner to move the second one (`if (pm.balance < minBalance)`) up so we don't have to repeat ourselves

Reply:
> maybe we can try adding filter: grayscale(1); on the PaymentEntryContainer when disabled?

I just tried this, no effect.

Reply:
Right, it actually worked, my eyes missed the change in the card icon. 😄

Reply:
`css={disabled? 'filter: grayscale(1);' : undefined}`

Is it okay to add it to prop this way?

Reply:
yep :slightly_smiling_face: